app/routes.py

```diff
@@ -1,2 +1,47 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, abort
 
+
+class Planet:
+    def __init__(self, id, name, description, gravity):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.gravity = gravity
+
+planets = [
+    Planet(1, 'Mercury', 'The closest planet to the sun! REALLY HOT!', '3.7 m/s2'),
+    Planet(2, 'Venus',  'Another hot planet! Actually hotter than Mercury!', '8.87 m/s2'),
+    Planet(3, 'Earth', 'Third Planet from the Sun. Maybe a little special. Much colder than the first two.', '9.8 m/s2')
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["GET"])
+def handle_planets():
+    planets_result = []
+    for planet in planets:
+        planets_result.append(dict(
+            id = planet.id,
+            name = planet.name,
+            description = planet.description,
+            gravity = planet.gravity
+        ))
```

Comment:
this could be moved to your planets class as an instance method. Then you would call that instance method here.
```
 def make_dict(self):
        return dict(
            id=self.id,
            name=self.name,
            description=self.description,
            gravity=self.gravity,  
        )
```

Reply:
Yup. We should refactor it like that. Thank you for the suggestion.